src/bundler.ts

```diff
@@ -393,7 +393,7 @@ export class Bundler {
       ast: ASTNode,
       bundle: AssignedBundle,
       bundleManifest: BundleManifest) {
-    const visitedUrls = new Set<UrlString>();
+    const visitedUrls = new Set<UrlString>(bundle.bundle.stripImports);
```

Comment:
I've decided to rename that variable and the parameter for the inlining function to `stripImports` since it is closely tied to that use case anyway.